include/ocpp/common/websocket/websocket_tls_tpm.hpp

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2020 - 2023 Pionix GmbH and Contributors to EVerest
+#ifndef OCPP_WEBSOCKET_TLS_TPM_HPP
+#define OCPP_WEBSOCKET_TLS_TPM_HPP
+
+#include <ocpp/common/evse_security.hpp>
+#include <ocpp/common/websocket/websocket_base.hpp>
+
+#include <queue>
+namespace ocpp {
+
+struct ConnectionData;
+struct WebsocketMessage;
+
+/// @brief Experimental libwebsockets TLS connection
+class WebsocketTlsTPM final : public WebsocketBase {
```

Comment:
Inheriting from WebsocketBase. It contains some unrelated data, should this be re-thought?